botfront/cypress/integration/webhooks/webhook_side_effects.spec.js

```diff
@@ -0,0 +1,31 @@
+/* global cy Cypress:true */
```

Comment:
Why not test that as part of trainAndDeploy spec?

Reply:
Side effects sounds weird to me in that context.

Reply:
I can, I moved it because I don't like having multiple describes in one cypress file and it's a waste to setup git integration for this

at the time I was thinking I would add tests for other webhooks but as there isn't currently a clean way to spoof the endpoint I decided not to